server/src/lexer/pest/python.pest

```diff
@@ -54,34 +54,37 @@ inline_comment_contents = { logical_line ~ newline_eoi }
 // [C standard, section 5.1.1.2](https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3220.pdf#page=24),
 // if a line of code ends with a backslash, it continues on the next line. This
 // is a logical line; treat it as a single line.
-logical_line = _{ ("\\" ~ NEWLINE | not_newline)* }
+logical_line = _{ ("//" ~ NEWLINE | not_newline)* }
 newline_eoi  = _{ NEWLINE | EOI }
 not_newline  = _{ !NEWLINE ~ ANY }
 
 // ## Block comments
 //
 // Block comments never combine.
-block_comment = {
-    white_space ~ block_comment_opening_delims ~ block_comment_pre ~ contents ~ " "? ~ "*/" ~ white_space ~ block_comment_ending
-}
+//block_comment = {
+//    white_space ~ block_comment_opening_delims ~ block_comment_pre ~ contents ~ " "? ~ "'''" ~ white_space ~ block_comment_ending
+//}
 // Support multiple opening delimiters for other languages; assume the closing
 // delimiter is always the same.
-block_comment_opening_delims  = _{ block_comment_opening_delim_0 }
-block_comment_opening_delim_0 =  { "/*" }
-block_comment_opening_delim_1 =  { "unused" }
-block_comment_opening_delim_2 =  { "unused" }
-block_comment_closing_delim   = _{ "*/" }
-block_comment_pre             =  { " " | newline_eoi }
+ //block_comment_opening_delims  = _{ block_comment_opening_delim_0 }
+ //block_comment_opening_delim_0 =  { "'''" }
+// block_comment_opening_delim_1 =  { "unused" }
+// block_comment_opening_delim_2 =  { "unused" }
+// block_comment_closing_delim   = _{ "'''" }
+// block_comment_pre             =  { " " | newline_eoi }
 // TODO: rewrite for languages that supported nested block comments.
 //
 // If there's a space immediately before the block closing delimiter, don't
 // capture it, so that this space can be discarded in the `block_comment` token.
-contents = { (!(" "? ~ block_comment_closing_delim) ~ ANY)* }
+// contents = { (!(" "? ~ block_comment_closing_delim) ~ ANY)* }
 // Use this so that the ending (usually a newline) gets captured in a token.
-block_comment_ending = { newline_eoi }
+// block_comment_ending = { newline_eoi }
 
 // ## Code blocks
-code_block = { code_line+ }
+
+multiline_string = { "'''" ~ (!"'''" ~ ANY)* ~ "'''" }
+root = _{ multiline_string | inline_comment }
+code_block = { multiline_string | code_line+ }
```

Comment:
Not the right approach -- a code block is:
- Not a doc block, followed by non-newline characters then a newline OR
- Not a doc block, followed by non-newline characters  then EOI OR
- non-newline characters followed by a newline OR
- a multi-line string